service/key/key_test.go

```diff
@@ -0,0 +1,55 @@
+package key
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/giantswarm/certificatetpr"
+)
+
+func Test_Organization_sort(t *testing.T) {
+	testCases := []struct {
+		Organizations         []string
+		ClusterComponent      string
+		ExpectedOrganizations []string
+	}{
+		{
+			Organizations:         []string{},
+			ClusterComponent:      "api",
+			ExpectedOrganizations: []string{"api"},
+		},
+		{
+			Organizations:         []string{"system:master"},
+			ClusterComponent:      "api",
+			ExpectedOrganizations: []string{"api", "system:master"},
+		},
+		{
+			Organizations:         []string{"system:master", "giantswarm"},
+			ClusterComponent:      "api",
+			ExpectedOrganizations: []string{"api", "system:master", "giantswarm"},
+		},
```

Comment:
Lets have another test with the same organizations but ordered the other way around.

Reply:
Do the tests actually break when the fix to the fixed function is removed?

Reply:
No, and I cannot reproduced it at all :(

Reply:
Sure, But i don't know how to reproduce it. 've created the same case in a dummy project ( with struct, method and so on. and cannot reproduce :/

Reply:
Does that mean when you deploy cert-operator with master everything is fine again?

Reply:
No, the bug is there. But cannot reproduce it in the test.

Reply:
I had the sort stuff there. but anyway it's not reproducing the bug we found